routes/api/follows.js

```diff
@@ -5,14 +5,14 @@ const { sequelize, Follow } = require("../../db/models");
 const {asyncHandler} = require('../utils');
 
 /* Delete a follower from a user */
-router.delete('/follows/:id', asyncHandler(async (req, res, next) => {
+router.delete('/follows/:id', restoreUser, asyncHandler(async (req, res, next) => {
```

Comment:
Make sure to import this middleware from `auth.js` in the root of the project.

Reply:
Middleware imported.